content/mooreslaw-tutorial.md

```diff
@@ -44,27 +44,24 @@ the 53 years following his prediction. You will determine the best-fit constants
 
 * NumPy
```

Comment:
unrelated to the PR, but it would be nice to also link numpy? (looks a bit weird in the rendered version that only mpl is linked)